js-exercises/animal-sort/animalSort.js

```diff
@@ -8,16 +8,10 @@ const animalSort = animals => {
   }
 
   return animals.sort((animalA, animalB) => {
-    if (animalA.numberOfLegs > animalB.numberOfLegs) {
-      return 1;
-    }
-    if (animalA.numberOfLegs === animalB.numberOfLegs) {
+   if (animalA.numberOfLegs === animalB.numberOfLegs) {
       return animalA.name.localeCompare(animalB.name);
     }
-    if (animalA.numberOfLegs < animalB.numberOfLegs) {
-      return -1;
-    }
-    return 1;
+   return animalA.numberOfLegs > animalB.numberOfLegs;
```

Comment:
oh sorry, it should be
```suggestion
   return animalA.numberOfLegs - animalB.numberOfLegs;
```

Reply:
done.